### ISSUE TITLE
Use the logging module for errors

### ISSUE THREAD

**Issue by kinow** [simonarvin/eyeloop#8]
A bit tricky to find out what went wrong where. I had some output in the console that I couldn't tell if it was normal logs, or errors.

It was a `print(e)` where `e` is an `Exception`. It would be helpful either being able to control what's displayed, or at least having the traceback in the console output. The `logging` module could be useful here, though there are probably many other ways to fix it.

**Comment by cfculhane** [comment]
+1 on this too, I've got a logging module and config I've used for many projects in the past that handles this stuff nicely, I'll add it once the restructure has been completed

**Comment by cfculhane** [comment]
I've made good progress on this, have implemented logging. This will be a large messy commit with lots of the print() statements converted into logging.

**Comment by kinow** [comment]
Happy to help with testing/reviewing if necessary @cfculhane ! Thanks :tada: !

**Comment by cfculhane** [comment]
Will definitely need a second pair of eyes on the PR, its one of refactoring things where you add one thing, realise something else has broken/exposed something else as broken, then you create a nice little rabbit hole to fall down, meaning the code diff will be a bit hard to follow. 

I'm new to contributing to open source projects via PRs, but I'll try and split the commits into smaller, functional units so that you can follow my thinking
